tools/linter.py

```diff
@@ -15,6 +15,7 @@
 EXCLUDE = (
     "numpy/typing/tests/data/",
     "numpy/__config__.py",
+    "numpy/f2py",
```

Comment:
RE: This seems rather harsh, but the `f2py` code has many lines which fail the linter; and this is the cleanest way to deal with this for now.

Reply:
Seems fair. I wish there was a python formatter that did what we want.

Reply:
We did talk about running a linter through all the code at some point. Maybe we can do that in a subsequent PR